python/setup.cfg

```diff
@@ -45,6 +45,8 @@ python_requires = >=3.8
 install_requires =
     mmh3
 [options.extras_require]
+boto =
```

Comment:
This creates an optional profile, like `iceberg[boto]` right?

Reply:
Yep, exactly